exporter/streams_test.go

```diff
@@ -80,15 +81,24 @@ func TestGetStreamInfo(t *testing.T) {
 			if isNotTestTimestamp(info.LastGeneratedId) {
 				t.Errorf("Stream LastGeneratedId mismatch.\nActual: %#v;\nExpected any of: %#v\n", info.LastGeneratedId, TestStreamTimestamps)
 			}
+			if info.FirstEntryId != TestStreamTimestamps[0] {
+				t.Errorf("Stream FirstEntryId mismatch.\nActual: %#v;\nExpected any of: %#v\n", info.FirstEntryId, TestStreamTimestamps)
+			}
+			if info.LastEntryId != TestStreamTimestamps[len(TestStreamTimestamps)-1] {
+				t.Errorf("Stream LastEntryId mismatch.\nActual: %#v;\nExpected any of: %#v\n", info.LastEntryId, TestStreamTimestamps)
+			}
+			if info.MaxDeletedEntryId != "0-0" {
+				t.Errorf("Stream MaxDeletedEntryId mismatch.\nActual: %#v;\nExpected: %#v\n", info.MaxDeletedEntryId, "0-0")
+			}
 		})
 	}
 }
 
 func TestScanStreamGroups(t *testing.T) {
-	if os.Getenv("TEST_REDIS_URI") == "" {
-		t.Skipf("TEST_REDIS_URI not set - skipping")
+	if os.Getenv("TEST_REDIS7_URI") == "" {
```

Comment:
Is it worth breaking this out into a separate test that only tests the new redis7 metrics (and uses the redis7 instance?)

Reply:
That might be a better case since then we can test backwards compatibility as well as redis 7 exposing new metrics, I'll adjust this in the mean time to test using redis 6 and have additional redis 7 tests.

Reply:
Included redis 7 specific tests for `TestGetStreamInfo` & `TestScanStreamGroups`.

Reply:
Thanks!